tests/build/chains.go

```diff
@@ -274,7 +338,31 @@ var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", Label("ec", "HA
 				printTaskRunStatus(tr, namespace, *fwk.AsKubeAdmin.CommonController)
 				GinkgoWriter.Printf("Make sure TaskRun %s of PipelineRun %s failed\n", tr.PipelineTaskName, pr.Name)
 				Expect(tekton.DidTaskSucceed(tr)).To(BeFalse())
-				// Because the task fails, no results are created
+
+				// Verify the logs of failed pipeline are stored in the persistent storage
+				tekton.DeleteResultTestToken(fwk.AsKubeAdmin.CommonController, pr.Namespace)
+				token, err := tekton.CreateResultTestToken(fwk.AsKubeAdmin.CommonController, pr.Namespace)
+				Expect(err).NotTo(HaveOccurred())
+
+				// Retrive Result REST API url
+				resultRoute, err := fwk.AsKubeAdmin.CommonController.GetOpenshiftRoute("tekton-results", "tekton-results")
+				Expect(err).NotTo(HaveOccurred())
+				resultUrl := fmt.Sprintf("https://%s", resultRoute.Spec.Host)
+				resultClient := pipeline.NewClient(resultUrl, token)
+
+				// Verify if result is stored in Database
+				logs, err := resultClient.GetLogs(pr.Namespace, string(pr.GetUID()))
+				Expect(err).NotTo(HaveOccurred())
+				Expect(len(logs.Record)).NotTo(BeZero(), "No logs found for PipelineRun %s", pr.Name)
+				for log := range logs.Record {
+					fmt.Println("log: ", logs.Record[log].Name)
+				}
+
+				// Verify if result is stored in S3
+				log, err := resultClient.GetLogByName(logs.Record[0].Name)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(len(log)).NotTo(BeZero(), "No log content found for PipelineRun %s", pr.Name)
+				Expect(log).To(ContainSubstring("PipelineRun name from params:d"))
```

Comment:
isn't this a duplication of what we already have [here](https://github.com/redhat-appstudio/e2e-tests/pull/462/files#diff-e79e8a91dc80ce7b323052d8b4e364284a5cfd76e9ea6ede9dde9299078545e7R251-R312)?

Reply:
yes, here is to ensure the logs of failed pipeline can be stored